src/main/resources/hudson/plugins/gradle/injection/InjectionConfig/config.jelly

```diff
@@ -1,17 +1,27 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:l="/lib/layout">
     <f:section title="${%Gradle Enterprise integration}">
+        <f:optionalBlock field="enabled" title="${%Enable auto-injection}" inline="true">
 
-        <j:if test="${instance.showLegacyConfigurationWarning}">
-            <f:entry>
-                <div class="alert alert-danger">
-                    <l:icon class="icon-error icon-sm" alt="${%Unsupported configuration detected}"/>
-                    An unsupported configuration has been detected. Please refer to the following <a href="https://github.com/jenkinsci/gradle-plugin#20">migration guide</a> on how to upgrade it.
-                </div>
-            </f:entry>
-        </j:if>
+            <j:if test="${instance.showLegacyConfigurationWarning}">
+                <f:entry>
+                    <div class="alert alert-danger">
+                        <l:icon class="icon-error icon-sm" alt="${%Unsupported configuration detected}"/>
+                        An unsupported configuration has been detected. Please refer to the following <a href="https://github.com/jenkinsci/gradle-plugin#20">migration guide</a> on how to upgrade it.
+                    </div>
+                </f:entry>
+            </j:if>
 
-        <f:optionalBlock field="enabled" title="${%Enable auto-injection}" inline="true">
+            <j:if test="${instance.unsupportedMavenPluginWarningDetails != null}">
+                <f:entry>
+                    <div class="alert alert-danger">
+                        <l:icon class="icon-error icon-sm" alt="${%Unsupported Maven Integration plugin version detected}"/>
+                        Detected Maven Integration plugin version ${instance.unsupportedMavenPluginWarningDetails.mavenPluginVersion}.
+                        For auto-injection of the Gradle Enterprise Maven extensions, version ${instance.unsupportedMavenPluginWarningDetails.minimumSupportedVersion} is required.
```

Comment:
Extensions is plural.

Reply:
It was plural is your comment [here](https://github.com/jenkinsci/gradle-plugin/pull/214#discussion_r1019660221), so I think @c00ler just posted it as is.

Reply:
Good catch! Yes, I just copied it. Will update.

Reply:
Done

Reply:
Right my bad. Can we use the following then on both places @c00ler ?

```
Detected Maven Integration plugin version x.x.x. For auto-injection of the Gradle Enterprise Maven extension, version x.x.x and above is required. Please upgrade the version of the Maven Integration plugin.
```